src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+      class="card__img"
+      src="images/imac.jpeg"
+      alt="img"
+      >
+
+      <div class="card__inform">
+        <div class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+        <div class="card__product-code">Product code: 195434</div>
```

Comment:
Do not use div for text content. Use semantic tags instead
Fix all similar cases